tools/ColorTool/ColorTool/IniSchemeParser.cs

```diff
@@ -85,6 +86,10 @@ public ColorScheme ParseScheme(string schemeName, bool reportErrors = false)
 
             string[] tableStrings = new string[COLOR_TABLE_SIZE];
             uint[] colorTable = null;
+            uint? forgroundColor = null;
```

Comment:
Can you fix the typo here? It should be "foregroundColor" here and for all references (as well as "popupForegroundColor" below).